freqtrade/tests/data/test_dataprovider.py

```diff
@@ -51,6 +51,39 @@ def test_historic_ohlcv(mocker, default_conf, ticker_history):
     assert historymock.call_args_list[0][1]["ticker_interval"] == "5m"
 
 
+def test_get_pair_dataframe(mocker, default_conf, ticker_history):
+    default_conf["runmode"] = RunMode.DRY_RUN
+    ticker_interval = default_conf["ticker_interval"]
+    exchange = get_patched_exchange(mocker, default_conf)
+    exchange._klines[("XRP/BTC", ticker_interval)] = ticker_history
+    exchange._klines[("UNITTEST/BTC", ticker_interval)] = ticker_history
+    dp = DataProvider(default_conf, exchange)
```

Comment:
Just for readability - can you add in a space between setup-code and where the actual tests start?

right before this line, basically.

Reply:
improved it somehow, done.